search-app/src/cmr/search/services/variable_service.clj

```diff
@@ -210,3 +211,11 @@
   "Dissociates a variable from the given list of variable associations in json."
   [context variable-name variable-associations-json]
   (link-variable-to-collections context variable-name variable-associations-json :delete))
+
+(defn search-for-variables
+  "Returns the variable search result in JSON format."
+  [context params]
+  (let [results (:results (query-service/find-concepts-by-parameters
+                            context :variable (assoc params :result-format :json)))]
+    (-> (json/parse-string results true)
```

Comment:
But you can have `results` being the first arg to `(->`, and then you'd be good.